### DISCUSSION TITLE
Making C-j and C-k work through ideavimrc ?

### DISCUSSION THREAD

**Discussion by asmodeus812** [JetBrains/ideavim#443] [Q&A]
Hello i am trying to make the regular c-j and c-k bindings work in intellij to scroll context menus, dialogs and popups. I did find that if you remap them in keymap in settings and bind them to up and down seems to work. But i would like to avoid having additional custom settings (and version control the entire intellij set of settings) like that outside of the vimrc. I was wondering if there is an api that allows us to do an intellij binding through the ideavimrc. Or any other way to make it work from the rc file. Thanks !

**Comment by AlexPl292** [comment]
Hi, unfortunately, at the moment IdeaVim works only in the editor and a bit in the project tree. I believe this issue fits the most for your case, you can vote for it (we prioritise the issues based on the votes): https://youtrack.jetbrains.com/issue/VIM-2323
